src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import '../utils/variables';
```

Comment:
Ensure that the `variables` file contains definitions for all the color variables used in your SCSS files, such as `$color-white`, `$text-main-color`, `$text-secondary-color`, and `$color-blue-accent`.